securedrop_client/gui/main.py

```diff
@@ -172,9 +172,15 @@ def on_source_changed(self):
         """
         source_item = self.main_view.source_list.currentItem()
         source_widget = self.main_view.source_list.itemWidget(source_item)
-        if source_widget:
+
+        # Show conversation for the currently-selected source if it hasn't been deleted. If the
+        # current source no longer exists, clear the conversation for that source
+        self.current_source = None
```

Comment:
do we want to set `self.current_source = None` here? i think it makes sense to set this during the `else` block when we clear the conversation?

Reply:
that's a good change. i think we could even go further and delete the member `current_source` from the client main window since we aren't using it anywhere

Reply:
oh huh! if that's true let's def excise it